debezium-core/src/main/java/io/debezium/pipeline/source/snapshot/incremental/AbstractIncrementalSnapshotChangeEventSource.java

```diff
@@ -302,8 +303,8 @@ protected void readChunk(P partition, OffsetContext offsetContext) throws Interr
                         continue;
                     }
                     if (LOGGER.isInfoEnabled()) {
-                        LOGGER.info("Incremental snapshot for table '{}' will end at position {}", currentTableId,
-                                context.maximumKey().orElse(new Object[0]));
+                        Loggings.logInfoAndTraceRecord(LOGGER, context.maximumKey().orElse(new Object[0]),
+                                "Incremental snapshot for table '{}' will end at position", currentTableId);
```

Comment:
It looks like the end position is not getting logged after this change. 
Here is the log from one of the tests where I noticed it:
`2025-03-21 09:24:15,874 INFO   Postgres|test_server|0|streaming|postgres  Incremental snapshot for table 's1.part2' will end at position   [io.debezium.pipeline.source.snapshot.incremental.AbstractIncrementalSnapshotChangeEventSource]
`

Reply:
@b-goyal I believe the wording of this message needs to be updated or maybe it will need to be altogether moved to a diffeent log level.
@kgalieva You are right, OTH (and I admit I never thougth about that) this really can be potential leak of data into logs.

Reply:
Thanks @jpechane and @kgalieva , I will raise a followup PR with the fix.

Reply:
A PR to improve this log line is raised here : https://github.com/debezium/debezium/pull/6247. Thanks!